Makefile

```diff
@@ -159,7 +159,7 @@ rebuild_client_start: docker_check ## Rebuild and run a client daemon which is o
 
 .PHONY: client_connect
 client_connect: docker_check ## Connect to the running client debugging daemon
-	docker exec -it client /bin/bash -c "POCKET_P2P_IS_CLIENT_ONLY=true go run -tags=debug app/client/*.go debug --remote_cli_url=http://validator1:50832"
+	docker exec -it client /bin/bash -c "POCKET_REMOTE_CLI_URL=http://validator1:50832 go run -tags=debug app/client/*.go debugui"
```

Comment:
:100:

Reply:
Can you elaborate on why remove `POCKET_P2P_IS_CLIENT_ONLY`?

Reply:
> Is there a reason not to move this into the [docker-compose.yaml](https://github.com/pokt-network/pocket/blob/main/build/deployments/docker-compose.yaml?rgh-link-date=2023-07-13T06%3A39%3A54Z#L24).

Nope.

> It looks like it's already set there, wdyt about just removing it here and relying on the env var in docker-compose.yaml?

I prefer this as well. Will give it a shot and report

> Can you elaborate on why remove POCKET_P2P_IS_CLIENT_ONLY?

Unless you have reason not to, I SHOULD remove it from `P2PConfig` in this PR as well. I do not believe it's being used anywhere.

Given that we have this function which is unrelated:

```go
func (m *p2pModule) isClientDebugMode() bool {
	return m.GetBus().GetRuntimeMgr().GetConfig().ClientDebugMode
}
```

Reply:
That helper was just there to support the existing functionality but I don't think that config is a dependency of any necessary logic.